pkg/bootkube/status.go

```diff
@@ -35,7 +36,7 @@ func WaitUntilPodsRunning(pods []string, timeout time.Duration) error {
 		return fmt.Errorf("error while checking pod status: %v", err)
 	}
 
-	UserOutput("All self-hosted control plane components successfully started\n")
+	UserOutput("All self-hosted control plane and checkpointing components successfully started\n")
```

Comment:
nit: For this and the change below, I don't think we really need to make a distinction between component types in the output during the installation.